src/About/Feedback.tsx

```diff
@@ -124,6 +132,11 @@ const FeedbackDialog = ({
                                     value={feedback}
                                     onChange={e => setFeedback(e.target.value)}
                                 />
+                                {errorMessage ? (
+                                    <p className="tw-mb-0 tw-mt-1 tw-text-right tw-text-xs tw-text-red">
+                                        {errorMessage}
+                                    </p>
+                                ) : null}
```

Comment:
Maybe just me but I feel like a ternary expression makes this a bit harder to read.
```suggestion
                                {!!errorMessage && (
                                    <p className="tw-mb-0 tw-mt-1 tw-text-right tw-text-xs tw-text-red">
                                        {errorMessage}
                                    </p>
                                )}
```

Reply:
I do this way because some engines can produce bugs because of expression rendering false / undefined.
Remix JS and React Native have this issue so I better be safe :) Our setup seems ok, but that's more as a new habit.

With the readability part I agree

Reply:
Instead of Mias double negation, a simple negation and `||` should also work:

```suggestion
                                {!errorMessage || (
                                    <p className="tw-mb-0 tw-mt-1 tw-text-right tw-text-xs tw-text-red">
                                        {errorMessage}
                                    </p>
                                )}
```
(In my head I am reading this as “Either there is no error message or you output the error message”)

But I am fine with all three variants.